api/ws_api.go

```diff
@@ -26,13 +26,27 @@ var (
 	}
 )
 
+var upgrader = websocket.Upgrader{
+	// good average time since this is not a high-latency operation such as video streaming
+	HandshakeTimeout: time.Second * 5,
+
+	// probably more that enough but this is a good average size
+	ReadBufferSize:  2048,
+	WriteBufferSize: 2048,
+	CheckOrigin:     func(r *http.Request) bool { return true },
+}
+
+type ServerOptions struct {
+	Port  *int
+	Db    *sql.DB
+	Stage string
+}
+
 type Server struct {
-	Port     *int
-	Upgrader websocket.Upgrader
-	Db       *sql.DB
-	Games    map[string]*md.Game
-	Players  map[string]*md.Player
-	mu       sync.RWMutex
+	Db      *sql.DB
```

Comment:
Do we still need this `Db` field?

Reply:
Yea we don't need that. after we want to set up the database, we actually separate the handlers and DB doesn't have to be in server struct